src/openrct2/scripting/ScTile.hpp

```diff
@@ -0,0 +1,1112 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2018 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#ifdef __ENABLE_SCRIPTING__
+
+#    include "../Context.h"
+#    include "../common.h"
+#    include "../world/Footpath.h"
+#    include "../world/Scenery.h"
+#    include "../world/Sprite.h"
+#    include "../world/Surface.h"
+#    include "Duktape.hpp"
+#    include "ScriptEngine.h"
+
+#    include <cstdio>
+#    include <cstring>
+
+namespace OpenRCT2::Scripting
+{
+    class ScSurfaceElement;
+
+    class ScTileElement
```

Comment:
Class is using snake_cast instead of CamelCast.

Should be `ScriptTileElement`